Microsoft.Toolkit.Uwp/Helpers/ObjectStorage/BaseObjectStorageHelper.cs

```diff
@@ -21,14 +21,13 @@ public abstract class BaseObjectStorageHelper : IObjectStorageHelper
         /// <summary>
         /// Initializes a new instance of the <see cref="BaseObjectStorageHelper"/> class,
         /// which can read and write data using the provided <see cref="IObjectSerializer"/>;
-        /// if none is provided, a default Json serializer will be used (based on <see cref="DataContractSerializer"/>).
-        /// In 6.1 and older the default Serializer was based on Newtonsoft.Json and the new default Serializer may behave differently.
+        /// In 6.1 and older the default Serializer was based on Newtonsoft.Json.
         /// To implement a <see cref="IObjectSerializer"/> based on Newtonsoft.Json or System.Text.Json see https://aka.ms/wct/storagehelper-migration
```

Comment:
I think we forgot to make this like `LocalObjectStorageHelper`
```
/// In 6.1 and older the default Serializer was based on Newtonsoft.Json.
/// To implement an <see cref="IObjectSerializer"/> based on System.Text.Json, Newtonsoft.Json, or DataContractJsonSerializer see https://aka.ms/wct/storagehelper-migration
```

Reply:
Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/commit/bac5096aaccda47ea505c94710d04472061a355d